Source/UI/NewUI/BillingForm/View/BillingFormPhoneNumberText.swift

```diff
@@ -3,32 +3,41 @@ import PhoneNumberKit
 
 protocol BillingFormPhoneNumberTextDelegate {
     func updateCountryCode(code:Int)
+    func isInValidNumber()
 }
 
 final class BillingFormPhoneNumberText: BillingFormTextField {
+
+    // MARK: - Properties
+
     var phoneNumberTextDelegate: BillingFormPhoneNumberTextDelegate?
     var phoneNumber: PhoneNumber?
+    let phoneNumberKit = PhoneNumberKit()
 
+    var partialFormatter: PartialFormatter {
+       PartialFormatter(phoneNumberKit: phoneNumberKit, defaultRegion: "GB", withPrefix: true)
```

Comment:
Formatters are expensive operations, we surely don't want to create a new one every time we ask it something. Also, private?

Reply:
please suggest code using <cmd+g>

Reply:
This is old code that needs to be refactored. 
a new [Jira ticket](https://checkout.atlassian.net/browse/PIMOB-1230?atlOrigin=eyJpIjoiOGIwMGVhNmY1N2FmNGY4ZGI5ODY4ZGJiZjYwNDg2M2YiLCJwIjoiaiJ9) for it.